index.html

```diff
@@ -10,8 +10,52 @@
     <!-- Add a link to your CSS file here (use the line above to guide you) -->
     <link rel="stylesheet" href="css/style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
 </head>
 <body>
+<header>
+    <img class="logo" src="img/karma-logo.svg" alt="karma logo">
+    <nav>
+        <ul>
+            <li>Meet Karma</li>
+            <li>How It Works</li>
+            <li>Store</li>
+            <li>Blog</li>
+            <li>Help</li>
+            <li>Login</li>
+        </ul>
+    </nav>
+</header>
+
+<section class="banner">
+    <img class="first-banner" src="img/first-background.jpg" alt="first banner">
```

Comment:
As mentioned above, remove this img tag and set the image to be the `background-image` of the `.banner` in your CSS instead. 
This will help you re-position the h1 and h2 which are currently off-center.